docs/zenith/quickstart/index.md

```diff
@@ -0,0 +1,12 @@
+---
+slug: /zenith/quickstart/index
+displayed_sidebar: "zenith"
+---
+
+# Quickstart
+
+{@include: ../partials/_experimental.md}
+
+User quickstart
+
+(should this go under the "Using Dagger" menu?)
```

Comment:
Should this go under the "Using Dagger" menu?

Yes: it's a user quickstart
No: having it at the top-level is important, because this is the entrypoint for new users and should be easy to find 

Reviewers, wdyt?